dd-sdk-android-androidx/README.md

```diff
@@ -1,30 +1,41 @@
-# Datadog Integration to be able to use the TrackFragmentsAsViewsStrategy for AndroidX
+# Datadog extra tracking features for the Android Jetpack components.
```

Comment:
I wouldn't mention the tracking features here, as we may add stuff for tracing or logs or …

> Datadog SDK extra features for the Android Jetpack components.